backend/spec/controllers/spree/admin/products_controller_spec.rb

```diff
@@ -143,6 +143,60 @@
     end
   end
 
+  describe "updating variant image rules" do
```

Comment:
i assume this is because it's a spike, but would be interested in some failure cases here.